extensions/parquet/base/build.gradle

```diff
@@ -12,6 +12,7 @@ dependencies {
     implementation project(':Base')
     implementation project(':Util')
     implementation depCommonsIo
+    implementation project(path: ':engine-table') // TODO Look for a better way to do this
```

Comment:
We need help from more experienced `gradle`ers to decide if this is acceptable or there is a better solution.